release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2550,6 +2602,141 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2042596[*BZ#2042596*])
+
+* The alpha channel for the PTP Operator reports an error that prevents a `PtpConfig` CR from being applied when the `summary_interval` value does not match the value set for `logSyncInterval`. To work around this issue, update all reference PTP configurations to set `summary_interval` and `logSyncInterval` to -4.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2040799[*BZ#2040799*])
+
+* When you create spoke clusters using policies and the OpenShift GitOps Operator v1.4.1 on a {rh-rhacm-first} hub cluster, clusters and policies apps report errors and do not recover.
++
+To work around this issue, pin the OpenShift GitOps Operator to v1.3.2. To do this, make the following changes on the hub cluster:
++
+. Apply the following `Subscription` CR to the hub cluster:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-gitops-operator
+  namespace: openshift-operators
+spec:
+  name: openshift-gitops-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: openshift-gitops-operator.v1.3.2
+  installPlanApproval: Manual
+----
++
+. Optional: Manually approve the `installplan` created for the subscription:
++
+[source,terminal]
+----
+$ oc get installplan -n openshift-operators
+----
++
+.Expected output
+[source,terminal]
+----
+NAME            CSV                                APPROVAL   APPROVED
+install-7wdfv   openshift-gitops-operator.v1.3.2   Manual     false
+----
++
+.. Edit the `installplan` that has CSV version `v.1.3.2`:
++
+[source,terminal]
+----
+$ oc edit installplan -n openshift-operators install-7wdfv
+----
++
+.. Set the `approved` field to `true`:
++
+[source,yaml]
+----
+spec:
+  approval: Manual
+  approved: true
+  clusterServiceVersionNames:
+  - openshift-gitops-operator.v1.3.2
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2046775[*BZ#2046775*])
+
+* Sometimes, when provisioning spoke clusters, ArgoCD applications with 100 or more `Siteconfig` CRs can cause `OOMKilled` memory errors on the hub cluster, even when the `openshift-gitops-application-controller-0` is allocated 40GB memory.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2026476[*BZ#2026476*])
+
+* When deploying a multi-node cluster on bare-metal nodes, the machine config pool (MCP) adds an additional CRI-O drop-in that circumvents the link:https://github.com/openshift-kni/cnf-features-deploy/blob/master/cnf-tests/README.md#container-mount-namespace[container mount namespace drop-in]. This results in CRI-O being in the base namespace while the kubelet is in the hidden namespace. All containers fail to get any kubelet-mounted filesystems, such as secrets and tokens.
```

Comment:
Following up from #41683 - This looks good to me!

Reply:
Got /lgtm from QE colleague Joshua Clarke via Slack